scripts/mesoscale/plots/exevs_mesoscale_atmos_sref_cape_plots.sh

```diff
@@ -241,6 +240,10 @@ done
 
 tar -cvf evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar *.png
 
+if [ $SENDDBN = YES ] ; then
+     $DBNROOT/bin/dbn_alert MODEL EVS_RZDM $job $COMOUT/evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar
```

Comment:
I'm not sure that this is the correct path to this .tar file based on what you have in the SENDCOM block. Shouldn't it be $COMOUT/$STEP/$COMPONENT/$RUN.$VDATE/evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar?

Can you please check these paths in your other SENDDBN blocks as well?